helm_deploy/digital-prison-services/values.yaml

```diff
@@ -1,12 +1,69 @@
 ---
 # Values here are the same across all environments
+generic-service:
+  nameOverride: digital-prison-services
 
+  image:
+    repository: quay.io/hmpps/digital-prison-services
+    tag: latest
+    pullPolicy: IfNotPresent
+    port: 3000
+
+  ingress:
+    enabled: true
+    v1_2_enabled: true
+    v0_47_enabled: false
```

Comment:
For no downtime I thought we would want to enable both ingress controllers? Then once `v1_2` is serving traffic we can disable `v0_47`?

Reply:
As discussed we are not enabling `v0_47` because DPS has `ingress.yaml` for the current ingress, and we're enabling `v1_2`